src/containers/News.tsx

```diff
@@ -239,6 +272,25 @@ export default function (props: NewsProps): React.ReactElement<NewsProps> {
                   </div>
                 );
                 output.push(element);
+                if announce.image_url.length {
```

Comment:
Does this compile?

Reply:
I'm 99.9% sure it does, lemme double check

Reply:
> I'm 99.9% sure it does, lemme double check

Yea its all g, works fine